### DISCUSSION TITLE
MAUI Commandline to build maui ios app for iPhoneSimulator without CodeSigning key on Mac machine

### DISCUSSION THREAD

**Discussion by muffadal53** [dotnet/maui#18796] [Q&A]
Hi Community,

What is the command-line to build .net maui iOS release build on Mac machine for iPhoneSimulator without adding any CodeSigning key to keychain on mac device? I have tried with below command line but it is asking for valid Code signing key which I think should not be required for iPhone simulator:

`dotnet build ${APP_PROJECT_DIR} --configuration Release --no-restore -f net7.0-ios /p:verbosity=quiet /p:Platform="iPhoneSimulator"`

 I am getting following error:

No valid iOS code signing keys found in keychain. You need to request a codesigning certificate from https://developer.apple.com/.

Awaiting help on this. Thanks in advance.

**Comment by Yu-Core** [comment]
https://github.com/xamarin/xamarin-macios/wiki/Project-file-properties#enablecodesigning
